components/src/core/EmptyState/EmptyState.tsx

```diff
@@ -24,7 +24,7 @@ export type EmptyStateProps = {
 const useStyles = makeStyles((theme) => ({
     root: {
         height: '100%',
-        color: Colors.gray[500],
+        color: Colors.black[500],
```

Comment:
We probably should chat as a group about when we use gray[500] and when we use black[500]

Reply:
We in fact need a color audit. I will talk about that today during the sprint planning